Test.cs

```diff
@@ -12,7 +14,12 @@ public static long calculate(long number)
                 throw new Exception("Number must be non-negative.");
             }
```

Comment:
The method name `GetBranchNama` contains a typo. Consider renaming it to `GetBranchName`.